src/common/low_precision_transformations/src/fold_fake_quantize.cpp

```diff
@@ -28,35 +32,86 @@ FoldFakeQuantizeTransformation::FoldFakeQuantizeTransformation(const Params& par
         return transform(*context, m);
     };
 
-    auto m = std::make_shared<ngraph::pattern::Matcher>(fakeQuantize, matcher_name);
+    auto m = std::make_shared<ngraph::pattern::Matcher>(matcher, matcher_name);
     this->register_matcher(m, callback);
 }
 
 bool FoldFakeQuantizeTransformation::transform(TransformationContext& context, ngraph::pattern::Matcher &m) {
     const auto fakeQuantize = ov::as_type_ptr<opset1::FakeQuantize>(m.get_match_root());
-    if (fakeQuantize == nullptr) {
-        return false;
-    }
+    if (fakeQuantize) {
+        const auto constOutputSrc = getConstOutput(fakeQuantize);
+        if (constOutputSrc) {
+            // since outputLow & outputHigh are same constant, the output
+            // can be built by broadcasting the outputLow/outputHigh
+            const auto data_shape_node = std::make_shared<opset1::ShapeOf>(fakeQuantize->input_value(0));
+            const auto resultConstant = std::make_shared<opset1::Broadcast>(constOutputSrc, data_shape_node);
+            replace_node(fakeQuantize, resultConstant);
+            std::cout << fakeQuantize->get_friendly_name() << " is folded as const" << std::endl;
+            return true;
+        }
+
+        if (!canBeTransformed(context, fakeQuantize)) {
+            return false;
+        }
 
-    if (!canBeTransformed(context, fakeQuantize)) {
-        return false;
+        const auto constantShape = fakeQuantize->input(1).get_partial_shape();
+        if (constantShape.is_dynamic()) {
+            return false;
+        }
+
+        std::shared_ptr<ngraph::Node> resultConstant = NetworkHelper::fold_fake_quantize(
+            fakeQuantize,
+            false,
+            ((constantShape.rank().get_length() >= 2) && (constantShape[1] != 1ul)) ? 1ul : 0ul);
+        if (ov::is_type<opset1::Constant>(resultConstant)) {
+            replace_node(fakeQuantize, resultConstant);
+            return true;
+        }
     }
 
-    const auto constantShape = fakeQuantize->input(1).get_partial_shape();
-    if (constantShape.is_dynamic()) {
-        return false;
+
+    const auto dequantMultiply = ov::as_type_ptr<opset1::Multiply>(m.get_match_root());
+    if (dequantMultiply) {
```

Comment:
the `dequantizeMultiply`  is actually a normal multiply, not necessarily generated by `FakeQuantizeDecompositionTransformation` , it may comes directly from onnx frontend when translate `DequantizeLinear` op, and it's not FQ, so it only has 2 inputs, right?